samples/osgi-bundles/org.wso2.carbon.uuf.sample.features-app.bundle/src/main/java/org/wso2/carbon/uuf/sample/featuresapp/bundle/Library.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.uuf.sample.featuresapp.bundle;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Library {
+    public Map<String, String> getBook(String id) {
+        Map<String, String> bookData = new HashMap<>();
+        bookData.put("id", id);
+        bookData.put("title", "Harry Potter and the Deathly Hallows");
+        bookData.put("author", "J. K. Rowling");
+        return bookData;
```

Comment:
Shall we create a new class `Book` and return a Book instance instead of Map. Also put a comment saying 'we are returning a sample book data'.